javascript/src/main/resources/javascript/ManifestationInfo.test.js

```diff
@@ -597,6 +599,174 @@ UnitTest.addFixture( "ManifestationInfo.getFullTitle", function() {
+        "title": "Den store djævlekrig",
+        "sequence": "1"
+    };
+
+    Assert.equalValue( "get series title from common or local ", ManifestationInfo.getSeries( commonData, localData ), expected );
+
+
+    var commonDataString =
+        '<ting:container' +
+        ' xmlns:ting="http://www.dbc.dk/ting"' +
+        ' xmlns:ac="http://biblstandard.dk/ac/namespace/"' +
+        ' xmlns:dc="http://purl.org/dc/elements/1.1/"' +
+        ' xmlns:dcterms="http://purl.org/dc/terms/"' +
+        ' xmlns:dkabm="http://biblstandard.dk/abm/namespace/dkabm/"' +
+        ' xmlns:dkdcplus="http://biblstandard.dk/abm/namespace/dkdcplus/"' +
+        ' xmlns:docbook="http://docbook.org/ns/docbook"' +
+        ' xmlns:oss="http://oss.dbc.dk/ns/osstypes"' +
+        ' xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">' +
+        '<dkabm:record>' +
+        '<ac:identifier>25912233|870970</ac:identifier>' +
+        '<ac:source>Bibliotekskatalog</ac:source>' +
+        '<dc:title>Djævelens lærling</dc:title>' +
+        '<dc:title xsi:type="dkdcplus:full">Djævelens lærling</dc:title>' +
+        '<dc:title xsi:type="dkdcplus:series">NOT THIS Den store djævlekrig ; 1</dc:title>' +
+        '<dc:creator xsi:type="dkdcplus:aut">Kenneth Bøgh Andersen</dc:creator>' +
+        '<dc:creator xsi:type="oss:sort">Bøgh Andersen, Kenneth</dc:creator>' +
+        '</dkabm:record>' +
+        '</ting:container>';
+    var localDataString = 
+        '<ting:localData' +
+        ' xmlns:ting="http://www.dbc.dk/ting"' +
+        ' xmlns:ac="http://biblstandard.dk/ac/namespace/"' +
+        ' xmlns:dc="http://purl.org/dc/elements/1.1/"' +
+        ' xmlns:dcterms="http://purl.org/dc/terms/"' +
+        ' xmlns:dkabm="http://biblstandard.dk/abm/namespace/dkabm/"' +
+        ' xmlns:dkdcplus="http://biblstandard.dk/abm/namespace/dkdcplus/"' +
+        ' xmlns:docbook="http://docbook.org/ns/docbook"' +
+        ' xmlns:oss="http://oss.dbc.dk/ns/osstypes"' +
+        ' xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">' +
+        '<dkabm:record>' +
+        '<ac:identifier>25912233|870970</ac:identifier>' +
+        '<ac:source>Bibliotekskatalog</ac:source>' +
+        '<dc:title>Djævelens lærling</dc:title>' +
+        '<dc:title xsi:type="dkdcplus:full">Djævelens lærling</dc:title>' +
+        '<dc:title xsi:type="dkdcplus:series">THIS Den store djævlekrig ; 1</dc:title>' +
+        '<dc:creator xsi:type="dkdcplus:aut">Kenneth Bøgh Andersen</dc:creator>' +
+        '<dc:creator xsi:type="oss:sort">Bøgh Andersen, Kenneth</dc:creator>' +
+        '</dkabm:record>' +
+        '</ting:localData>';
+
+    var commonData = XmlUtil.fromString( commonDataString );
+    var localData = XmlUtil.fromString( localDataString );
+
+    var expected = {
+        "title": "THIS Den store djævlekrig",
+        "sequence": "1"
+    };
+
+    Assert.equalValue( "get series title from common or local ", ManifestationInfo.getSeries( commonData, localData ), expected );
```

Comment:
også omdøb her f.eks. 'get series title from local when both common and local are present'